src/main/java/dev/blaauwendraad/masker/json/config/KeyMaskingConfig.java

```diff
@@ -61,6 +62,7 @@ public String toString() {
                 .formatted(maskStringsWith, maskNumbersWith, maskBooleansWith);
     }
 
+    @NullUnmarked
```

Comment:
Why exactly do we mix NullUnmarked and NullMarked (on packages)? Shouldn't we have `Nullunmarked` everywhere? I think I'm missing something

Reply:
We can make the builder methods non-nullable, right?

Reply:
Okay, it actually did revert all the methods, so I actually marked the fields nullable instead 👍 Nice catch